zerver/openapi/zulip.yaml

```diff
@@ -1504,6 +1504,7 @@ paths:
           type: boolean
           default: None
         example: false
+        required: false
```

Comment:
This sort of fix to an adjacent bit of code should be its own commit.  It takes only <30 seconds to create a separate commit for this sort of thing if you know how to use `git rebase -i` properly.  And it's really valuable, because when a reviewer sees this change, they're going to wonder what it's doing here, which can add a significant amount of time to code review.